README.md

```diff
@@ -1,13 +1,11 @@
-# Reaction Storefront Next.js Starter Kit
+# Example Storefront
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Freactioncommerce%2Freaction-next-starterkit.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Freactioncommerce%2Freaction-next-starterkit?ref=badge_shield)
```

Comment:
Not sure how these badges work and how to update it, looking for some guidance

Reply:
Docs: https://docs.fossa.com/docs/automatic-updates#section-adding-a-fossa-badge-to-documentation-readme

Guessing you'll want to rename to match the repo rename but not 100% sure.